src/Evolution/Systems/GeneralizedHarmonic/GaugeSourceFunctions/DampedHarmonic.hpp

```diff
@@ -0,0 +1,277 @@
+
+namespace GeneralizedHarmonic {
+// @{
+/*!
+ * \ingroup GeneralizedHarmonic
+ * \brief Damped harmonic gauge source function.
+ *
+ * \details Computes the gauge source function designed for binary black hole
+ * evolutions. These have been taken from \cite Szilagyi2009qz and
+ * \cite Deppe2018uye.
+ *
+ * The covariant form of the source function \f$H_a\f$ is written as:
+ *
+ * \f{align*}
+ * H_a :=  [1 - R_{H_\mathrm{init}}(t)] H_a^\mathrm{init} +
+ *  [\mu_{L1} \mathrm{log}(\sqrt{g}/N) + \mu_{L2} \mathrm{log}(1/N)] t_a
+ *   - \mu_S g_{ai} N^i / N
+ * \f}
+ *
+ * where \f$N, N^k\f$ are the lapse and shift respectively,
+ * \f$t_a\f$ is the unit normal one-form to the spatial slice,
+ *
+ * \f{align*}
+ *  \mu_{L1} &= A_{L1} R_{L1}(t) W(x^i) \mathrm{log}(\sqrt{g}/N)^{e_{L1}}, \\
+ *  \mu_{L2} &= A_{L2} R_{L2}(t) W(x^i) \mathrm{log}(1/N)^{e_{L2}}, \\
+ *  \mu_{S} &= A_{S} R_{S}(t) W(x^i) \mathrm{log}(\sqrt{g}/N)^{e_{S}},
+ * \f}
+ *
+ * temporal roll-on functions \f$ R_X(t)\f$ (with label \f$ X\f$) are:
+ *
+ * \f{align*}
+ * \begin{array}{ll}
+ *     R_X(t) & = 1 - \exp[-((t - t_{0,X})/ \sigma_t^X)^4], & t\geq t_{0,X} \\
+ *     & = 0, & t< t_{0,X} \\
+ * \end{array}
+ * \f}
+ *
+ * and the spatial weight function is:
+ *
+ * \f{align*}
+ * W(x^i) = \exp[-(r/\sigma_r)^2].
+ * \f}
+ * This weight function can be written with multiple constant factors in the
+ * exponent in literature \cite Deppe2018uye, but we absorb them all into
+ * \f$ \sigma_r\f$ here.
+ *
+ * Note:
+ *   - Amplitude factors \f$ A_{X} \f$ are input as amp\_coef\_X
+ *   - \f$ R_{X} \f$ is specified by \f$\{ t_{0,X}, \sigma_t^X\}\f$, which are
+ *     input as \f$\{\f$t\_start\_X, sigma\_t\_X \f$\}\f$.
+ *   - Exponents \f$ e_X\f$ are input as exp\_X.
+ *   - The spatial weight function is specified completely by \f$\{\f$sigma\_r
+ * \f$\}\f$.
+ */
+template <size_t SpatialDim, typename Frame>
+struct DampedHarmonicHCompute : db::ComputeTag {
+  using argument_tags = tmpl::list<
+      Tags::InitialGaugeH<SpatialDim, Frame>, ::gr::Tags::Lapse<DataVector>,
+      ::gr::Tags::Shift<SpatialDim, Frame, DataVector>,
+      ::gr::Tags::SqrtDetSpatialMetric<DataVector>,
+      ::gr::Tags::SpacetimeMetric<SpatialDim, Frame, DataVector>, ::Tags::Time,
+      Tags::GaugeHRollOnStartTime, Tags::GaugeHRollOnTimeWindow,
+      ::Tags::Coordinates<SpatialDim, Frame>,
+      Tags::GaugeHSpatialWeightDecayWidth<Frame>>;
+
+  static typename db::item_type<Tags::GaugeH<SpatialDim, Frame>> function(
+      const typename db::item_type<Tags::InitialGaugeH<SpatialDim, Frame>>&
+          gauge_h_init,
+      const Scalar<DataVector>& lapse,
+      const tnsr::I<DataVector, SpatialDim, Frame>& shift,
+      const Scalar<DataVector>& sqrt_det_spatial_metric,
+      const tnsr::aa<DataVector, SpatialDim, Frame>& spacetime_metric,
+      double time, double t_start, double sigma_t,
+      const tnsr::I<DataVector, SpatialDim, Frame>& coords,
+      double sigma_r) noexcept;
```

Comment:
Please add a test for these compute items. I think even the `double`s have to be taken by const ref to avoid compiler errors, but note sure right now

Reply:
added